test/karma.conf.js

```diff
@@ -0,0 +1,77 @@
+var path = require('path');
+
+// allow running only certain directories
+var testDirs = [
```

Comment:
This code is just to make things easier to run directories, similar to opening up `localhost/test/checks`. This way you can pass `karma start test/karma.config.js testDirs=checks` and only run the check files (which is great as core files take the longest to run)